target/scripts/build/packages.sh

```diff
@@ -65,7 +65,7 @@ function _install_packages() {
     apt-transport-https bind9-dnsutils binutils bsd-mailx
     ca-certificates curl dbconfig-no-thanks
     dumb-init ed gnupg iproute2 iputils-ping
-    libdate-manip-perl libldap-common
+    less libdate-manip-perl libldap-common
```

Comment:
We have a few packages being introduced specifically for debugging, could we instead group these like we do with others to make that distinction clear?

`bind9-dnsutils` was added just for `dig` IIRC, and `iputils-ping` for `ping`.

Reply:
I can do that

Reply:
Done in [375d947](https://github.com/docker-mailserver/docker-mailserver/pull/3576/commits/375d947b762bd8726f1aa340c3b0ffc818eb5f8a)